osa/paths.py

```diff
@@ -102,8 +102,11 @@ def get_calibration_file(run_id: int) -> Path:
     """
     calib_dir = Path(cfg.get("LST1", "CALIB_DIR"))
     date = get_run_date(run_id)
-    file = calib_dir / date / f"pro/calibration_filters_52.Run{run_id:05d}.0000.h5"
-    return file.resolve()
+    file = calib_dir.glob(f"{date}/pro/calibration_filters_*.Run{run_id:05d}.0000.h5")
```

Comment:
For data processing purposes we always need the calibration file produced with filters 52. It might happen that other calibration files with different filters are produced, but we are not supposed to use them further in the analysis. So I think this function could remain the same

Reply:
ok, I will change it